docs/notebooks/des-gaia.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>"It is what you would get for the full catalog: " -&gt; This is what you would ge  for the full catalog: </p><p> </p><p> </p><p>It is a bit unclear the difference between two plots</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>